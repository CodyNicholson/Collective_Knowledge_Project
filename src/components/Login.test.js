//This file was begun using code from Link.react.test.js, from the Jest Testing React Apps Guide
//https://facebook.github.io/jest/docs/en/tutorial-react.html
import React from 'react';
import { Login } from './Login';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

var authorData = {username:"mikey", image:null};
var articleData = {author: authorData, createdAt: new Date(1997, 6, 24, 14, 39, 7)}

test('Tags module test', () => {
  const component = renderer.create(
    <MemoryRouter>
	    <Login email="testboi" password="testboipass"/>
    </MemoryRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});