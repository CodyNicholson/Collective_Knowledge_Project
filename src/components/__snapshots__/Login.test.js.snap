// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags module test 1`] = `
<div
  className="auth-page"
>
  <div
    className="container page"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6 offset-md-3 col-xs-12"
      >
        <h1
          className="text-xs-center"
        >
          Sign In
        </h1>
        <p
          className="text-xs-center"
        >
          <a
            href="/register"
            onClick={[Function]}
          >
            Need an account?
          </a>
        </p>
        <form
          onSubmit={[Function]}
        >
          <fieldset>
            <fieldset
              className="form-group"
            >
              <input
                className="form-control form-control-lg"
                onChange={[Function]}
                placeholder="Email"
                type="email"
                value="testboi"
              />
            </fieldset>
            <fieldset
              className="form-group"
            >
              <input
                className="form-control form-control-lg"
                onChange={[Function]}
                placeholder="Password"
                type="password"
                value="testboipass"
              />
            </fieldset>
            <button
              className="btn btn-lg btn-primary pull-xs-right"
              disabled={undefined}
              type="submit"
            >
              Sign in
            </button>
            <button
              className="btn btn-lg btn-primary pull-xs-left"
              disabled={true}
              onClick={[Function]}
              style={
                Object {
                  "opacity": 0.6,
                }
              }
              type="button"
            >
              Google Login
            </button>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
`;
